glusterd2/volume/store-utils.go

```diff
@@ -7,10 +7,13 @@ import (
 	"github.com/gluster/glusterd2/glusterd2/brick"
 	"github.com/gluster/glusterd2/glusterd2/store"
 	gderror "github.com/gluster/glusterd2/pkg/errors"
+	"github.com/gluster/glusterd2/pkg/utils"
 
 	"github.com/coreos/etcd/clientv3"
 	"github.com/pborman/uuid"
 	log "github.com/sirupsen/logrus"
+
```

Comment:
no need of extra line, as these packages are third-party libraries

Reply:
Sure, will remove it.